### PR TITLE
Ticket #8852 - convert archived to integer

### PR DESCRIPTION
**Pull request by melsawy**

### CODE REVIEW

File: test/unit/add_annotation.spec.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ import { mountWithIntl } from './helpers/intl-test';
 import AddAnnotation from '../../src/app/components/annotations/AddAnnotation';
 
 describe('<AddAnnotation />', () => {
-  const annotated_archived = { archived: true };
-  const annotated_not_archived = { archived: false };
+  const annotated_archived = { archived: 1 };
+  const annotated_not_archived = { archived: 0 };
```
Comment (alexmeedan) [inline test/unit/add_annotation.spec.js:8]: Is there a reason for not importing `CheckArchivedFlags` and using the constants from the tests too?

File: test/unit/create_task.spec.js [hunk 2]
```diff
@@ -4,13 +4,13 @@ import CreateTask from '../../src/app/components/task/CreateTask';
 
 
 describe('<CreateTask />', () => {
-  const media_archived = { archived: true , permissions:"{\"create Task\": true }" };
-  const media_not_archived = { archived: false, permissions:"{\"create Task\": true }" };
+  const media_archived = { archived: 1 , permissions:"{\"create Task\": true }" };
+  const media_not_archived = { archived: 0, permissions:"{\"create Task\": true }" };
```
Comment (alexmeedan) [inline test/unit/create_task.spec.js:8]: Is there a reason for not importing `CheckArchivedFlags` and using the constants from the tests too?

File: test/unit/web_page_media_card.spec.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ describe('<WebPageMediaCard />', () => {
       dbid: 10,
       quote: null,
       published: '1506728130',
-      archived: false,
+      archived: 0,
```
Comment (alexmeedan) [inline test/unit/web_page_media_card.spec.js:13]: Is there a reason for not importing `CheckArchivedFlags` and using the constants from the tests too?

File: test/unit/web_page_media_card.spec.js [hunk 4]
```diff
@@ -55,7 +55,7 @@ describe('<WebPageMediaCard />', () => {
       dbid: 26,
       quote: null,
       published: '1508470916',
-      archived: false,
+      archived: 0,
```
Comment (alexmeedan) [inline test/unit/web_page_media_card.spec.js:58]: Is there a reason for not importing `CheckArchivedFlags` and using the constants from the tests too?

File: test/unit/web_page_media_card.spec.js [hunk 5]
```diff
@@ -106,7 +106,7 @@ describe('<WebPageMediaCard />', () => {
       dbid: 10,
       quote: null,
       published: '1506728130',
-      archived: false,
+      archived: 0,
```
Comment (alexmeedan) [inline test/unit/web_page_media_card.spec.js:109]: Is there a reason for not importing `CheckArchivedFlags` and using the constants from the tests too?

**Review by alexmeedan — APPROVED** [review]
Looks great, Sawy! I'm just wondering why not use the constants file in the tests.

File: test/unit/web_page_media_card.spec.js (hunk 5 above)
Comment (melsawy) [reply test/unit/web_page_media_card.spec.js:109]: No as it's a test I used the value direct, will replace it with the constants

File: test/unit/add_annotation.spec.js (hunk 1 above)
Comment (melsawy) [reply test/unit/add_annotation.spec.js]: Fixed by using CheckArchivedFlags constants

File: test/unit/create_task.spec.js (hunk 2 above)
Comment (melsawy) [reply test/unit/create_task.spec.js]: Fixed by using CheckArchivedFlags constants

File: test/unit/web_page_media_card.spec.js (hunk 3 above)
Comment (melsawy) [reply test/unit/web_page_media_card.spec.js]: Fixed by using CheckArchivedFlags constants

File: test/unit/web_page_media_card.spec.js (hunk 4 above)
Comment (melsawy) [reply test/unit/web_page_media_card.spec.js]: Fixed by using CheckArchivedFlags constants

File: test/unit/web_page_media_card.spec.js (hunk 5 above)
Comment (melsawy) [reply test/unit/web_page_media_card.spec.js]: Fixed by using CheckArchivedFlags constants

**Review by caiosba — APPROVED** [review]

**Pull request merged**